system/lib/wasm_worker/library_wasm_worker.c

```diff
@@ -13,6 +13,8 @@
 #endif
 
 #define ROUND_UP(x, ALIGNMENT) (((x)+ALIGNMENT-1)&-ALIGNMENT)
+#define SBRK_ALIGN (__alignof__(max_align_t))
+#define STACK_ALIGN 16
```

Comment:
Isn't STACK_ALIGN also `(__alignof__(max_align_t))`?

Reply:
As it happens we use 16 byte alignment for the stack and 8 byte alignment for `__alignof__(max_align_t))`.

Perhaps we should lower our stack alignment since we recently lowers `__alignof__(max_align_t))` from 16 to 8, but that is outside the scope of this PR.